src/main/java/io/jenkins/plugins/coverage/CoverageChecksPublisher.java

```diff
@@ -0,0 +1,190 @@
+        for (String singleType : COVERAGE_TYPES) {
+            if (ratios.containsKey(singleType)) {
+                containedTypes.add(singleType);
+
+                float percentage = ratios.get(singleType);
+                coverages.append(String.format("%.2f%%|", percentage));
+
+                if (lastRatios.containsKey(singleType)) {
+                    float diff = percentage - lastRatios.get(singleType);
+                    trends.append(String.format("%+.2f%%", diff));
+
+                    if (Float.compare(diff, 0) > 0) {
+                        trends.append(" :arrow_up:|");
+                    } else if (Float.compare(diff, 0) < 0) {
+                        trends.append(" :arrow_down:|");
+                    } else {
+                        trends.append(" :arrow_right:|");
+                    }
+                } else {
+                    trends.append("-|");
+                }
+            }
+        }
+
+        return "||" + String.join("|", containedTypes) + "|\n"
+                + "|" + String.join("", Collections.nCopies(containedTypes.size() + 1, ":-:|")) + "\n"
+                + coverages + "\n"
+                + trends;
+    }
+
+    private String extractChecksTitle(final CoverageResult result) {
+        Map<CoverageElement, Ratio> lastRatios = getLastRatios(result);
+
+        String lineTitle;
+        float lineCoverage = result.getCoverage(CoverageElement.LINE).getPercentageFloat();
+        if (result.getLinkToBuildThatWasUsedForComparison() != null) {
+            lineTitle = extractChecksTitle("Line", "target branch build", lineCoverage,
+                    result.getChangeRequestCoverageDiffWithTargetBranch());
+        } else if (lastRatios.containsKey(CoverageElement.LINE)) {
+            lineTitle = extractChecksTitle("Line", "last successful build", lineCoverage,
+                    lineCoverage - lastRatios.get(CoverageElement.LINE).getPercentageFloat());
+        } else {
+            lineTitle = extractChecksTitle("Line", "", lineCoverage, 0);
+        }
+
+        String branchTitle;
+        float branchCoverage = result.getCoverage(CoverageElement.CONDITIONAL).getPercentageFloat();
+        if (lastRatios.containsKey(CoverageElement.CONDITIONAL)) {
+            branchTitle = extractChecksTitle("Branch", "last successful build", branchCoverage,
+                    branchCoverage - lastRatios.get(CoverageElement.CONDITIONAL).getPercentageFloat());
+        } else {
+            branchTitle = extractChecksTitle("Branch", "", branchCoverage, 0);
+        }
+
+        return lineTitle + " " + branchTitle;
+    }
+
+    private String extractChecksTitle(final String elementName, final String targetBuildName,
+                                      final float coverage, final float coverageDiff) {
+        StringBuilder title = new StringBuilder()
+                .append(elementName)
+                .append(" coverage: ")
```

Comment:
I would remove this line, it's obvious you're talking about coverage and github is truncating it in the UI
```suggestion
```